onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -521,13 +599,203 @@ def data_type(self):
 
     def is_tensor(self):
         '''
-        Returns True if the OrtValue is a Tensor, else returns False
+        Returns True if the OrtValue contains a Tensor, else returns False
         '''
         return self._ortvalue.is_tensor()
 
+    def is_sparse_tensor(self):
+        '''
+        Returns True if the OrtValue contains a SparseTensor, else returns False
+        '''
+        return self._ortvalue.is_sparse_tensor()
+
+    def is_tensor_sequence(self):
+        '''
+        Returns True if the OrtValue contains a Tensor Sequence, else returns False
+        '''
+        return self._ortvalue.is_tensor_sequence()
+
     def numpy(self):
         '''
         Returns a Numpy object from the OrtValue.
         Valid only for OrtValues holding Tensors. Throws for OrtValues holding non-Tensors.
+        Use accessors to gain a reference to non-Tensor objects such as SparseTensor
```

Comment:
So it is an error if this method is invoked for SparseTensors ?

Reply:
Yes, I am seeing that the code requires OrtValue to be a Tensor since numpy for us is a python mapping of our Tensor class.